openreview/conference/templates/areachairWebfield.js

```diff
@@ -505,17 +508,23 @@ var renderStatusTable = function(profiles, notes, allInvitations, completedRevie
   }
 };
 
-var updateReviewerContainer = function(paperNumber) {
+var updateReviewerContainer = function(paperNumber,renderEmptyDropdown=false) {
```

Comment:
same here, space after ','

Reply:
do we support default values in js? I didn't know it supports that.

Reply:
This is a feature of ES6. Since this code is running on the client side and needs to support IE11 we cannot use any ES6 features in webfield templates.

Reply:
looks like we stuck at es5 as long as we have the user defined webfields